src/ebfm/LOOP_EBM.py

```diff
@@ -14,7 +14,7 @@
 )
 from ebfm import LOOP_EBM_SWout, LOOP_EBM_insolation
 
-from coupler import Coupler
+from couplers import Coupler
```

Comment:
I could also avoid these breaking changes by introducing additional wrapper modules. But this feels like a bit too much.